src/plays/react-todo-app/ReactTodoApp.jsx

```diff
@@ -0,0 +1,84 @@
+import { getPlayById } from 'meta/play-meta-util';
+
+import PlayHeader from 'common/playlists/PlayHeader';
+
+import { useState, useEffect } from 'react';
+
+// css file
+import './react-todo.css';
+
+// components
+import Input from './components/Input';
+import TodoList from './components/TodoList';
+
+function ReactTodoApp(props) {
+  // Do not remove the below lines. 
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+
+  const [input, setInput] = useState("");
+  const [todos, setTodos] = useState([]);
+  const [status, setStatus] = useState("all");
+  const [filtered, setFiltered] = useState([]);
+
+  useEffect(() => {
+
+    const getTodos = () => {
+      if (localStorage.getItem("todos") === null) {
+        localStorage.setItem("todos", JSON.stringify([]));
```

Comment:
Change the localstorage key from `todos` to something unique, may be, append your github username to make it more unique.